Examples/Neutral_density_calculation.ipynb

```diff
@@ -0,0 +1,790 @@
+{
```

Comment:
<p>Instead of python comments, could we have markdown explanations before the code cell?</p><p> </p><p>(That's a general comment, not only for this cell!)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>